CONTRIBUTING.md

```diff
@@ -49,13 +49,13 @@ In general, contributors should develop on branches based off of `master` and pu
 
 ## Submitting a pull request
 
-0. [Fork][] and clone the repository
-0. Configure and install the dependencies: `make`
-0. Make sure the tests pass on your machine: `make test`
-0. Create a new branch based on `master`: `git checkout -b <my-branch-name> master`
-0. Make your change, add tests, and make sure the tests still pass
-0. Push to your fork and [submit a pull request][pr] from your branch to `master`
-0. Pat yourself on the back and wait for your pull request to be reviewed
+1. [Fork][] and clone the repository
```

Comment:
While I'm personally partial to zero-based indexing, I agree it's not intuitive to most people, and one-based numbering is in agreement with typical style guides, so I think this is a good way forward.